admin/dump_pdgid_to_lhcb.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
```

Comment:
Do write a header to explain what the file is for and what it produces.

Reply:
Added a docstring (plus copyright statement) in 38fda43